RELEASES.md

```diff
@@ -21,6 +21,41 @@ To summarize: **All our ibc-go releases allow chains to communicate successfully
 
 We ensure all major releases are supported by relayers ([hermes](https://github.com/informalsystems/ibc-rs), [rly](https://github.com/strangelove-ventures/relayer) and [ts-relayer](https://github.com/confio/ts-relayer) at the moment) which can relay between the new major release and older releases. We have no plans of upgrading to an IBC protocol specification v2.0, as this would be very disruptive to the ecosystem.
 
+## Stable Release Policy
+
+A Major Release series continues to receive bug fixes (released as a Patch Release) until it reaches End Of Life. A Major Release series reaches End of Life 6 months after the publication of the next Major Release series. For example, if the current Major Release series is v1, then v1 will be supported for up to 6 months after the publication of v2. 
```

Comment:
I personally would just use lowercase words, but I am not super strongly against otherwise...
```suggestion
A major release series continues to receive bug fixes (released as a patch release) until it reaches End Of Life. A major release series reaches End of Life 6 months after the publication of the next major release series. For example, if the current major release series is v1, then v1 will be supported for up to 6 months after the publication of v2. 
```